Dockerfile

```
@@ -9,5 +11,8 @@ RUN pip install -r /app/requirements.txt
 COPY ./src /app
 
 RUN echo "${BUILD_BRANCH}\n${BUILD_SHA}" > "/app/build_info.txt"
+ENV AbbotApiBaseUrl=https://app.ab.bot/api
+ENV HOST="0.0.0.0"
+ENV PORT="80"
 
```

Since we [default to `permissive` if `ABBOT_SANDBOX_POLICY` and `ABBOT_SANDBOXED` aren't specified](https://github.com/aseriousbiz/abbot-py/blob/e37e987239b3fde54173d539e921005a084a8840/src/SkillRunner/bot/policy.py#L6-L11), should we make that explicit here?

```suggestion
ENV ABBOT_SANDBOX_POLICY="permissive"

```